pages/common/svg2png.md

```diff
@@ -0,0 +1,32 @@
+# svg2png
+
+> Render an SVG image to a PNG image using cairo.
+> More information: <http://cairographics.org/>.
```

Comment:
Find me a direct link that instructs on the usage of this command.

Reply:
The latest release (0.1.3, 2005-06-15) of svg2png predates the project's use of git, unfortunately. In fact, most of the development actually predates the existence of git itself.

You can view the most recent snapshot [here](https://cairographics.org/snapshots/svg2png-0.1.3.tar.gz), and the relevant documentation is in `svg2png-0.1.3/src/args.h`. However, there is no available direct link to a project-hosted page that lists the arguments.

Reply:
Is this the same command or something else? https://manned.org/svg2png